include/boost/date_time/locale_config.hpp

```diff
@@ -22,7 +22,7 @@
 //This file basically becomes a noop if locales are not properly supported
 #if (defined(BOOST_NO_STD_LOCALE)  \
  || (BOOST_WORKAROUND( BOOST_MSVC, < 1300)) \
- || (BOOST_WORKAROUND( __BORLANDC__, BOOST_TESTED_AT( 0x581 )) ) \
+ || (BOOST_WORKAROUND( BOOST_BORLANDC, BOOST_TESTED_AT( 0x581 )) ) \
```

Comment:
so we're saying this compiler/std library still doesn't have std::locale?  Cause honestly I'd like to remove all of this at this point.   The support for MSVC <1300 for example is long gone in boost.

Reply:
The BOOST_BORLANDC stands for the old bcc32 compiler, not the current clang-based compiler(s). Feel free to remove all the old code referring to bcc32 if you like since I do not think the old bcc32 can compile current Boost code anyway. I have no interest in trying to get current Boost code to work with the old bcc32 compiler, and I do not test it. I am only trying to get current Boost to work with the much more compliant clang-based compilers.

Reply:
Ok thanks -- I'll probably clean some of this out in 1.74